test/esm/index.mjs

```diff
@@ -0,0 +1,60 @@
+import assert from "assert/strict";
+import { transformSync } from "@babel/core";
+
+import corejs2 from "babel-plugin-polyfill-corejs2";
+import corejs3 from "babel-plugin-polyfill-corejs3";
+import esShims from "babel-plugin-polyfill-es-shims";
+import regenerator from "babel-plugin-polyfill-regenerator";
+
+assert.equal(typeof corejs2, "function");
```

Comment:
This would be an object with `{ default: function, __esModule: true }` if we imported the CJS file.